plugins/inputs/elasticsearch_query/elasticsearch_query.go

```diff
@@ -0,0 +1,304 @@
+package elasticsearch_query
+
+import (
+	"context"
+	"fmt"
+	"log"
```

Comment:
You should _not_ use `log` but rather define 
```golang
Log telegraf.Logger `toml:"-"`
```
in your struct and the use the telegraf logging facility!